src/app/pages/main/main.component.ts

```diff
@@ -10,11 +11,17 @@ import { map } from 'rxjs/operators';
 })
 export class MainComponent {
   @Input()
-  communities: Communities;
+  /* communities: Communities; */
+  communitie$ = this.communityService.communities;
```

Comment:
This observable is going to perform two requests and more if we move to the new `faq` page and come back. We should avoid it using `shareReplay(1)`, but it can't be placed here. We should change service, and move `communities` getter to simple property (`communitie$`?). Something like
```ts
communitie$ = this.httpClient
      .get<Communities>(this.JSON_COMMUNITIES)
      .pipe(map(communities => this.normalizeCommunities(communities)), shareReplay(1));
```

Reply:
You are completely right, fixed in f4afebb